app/routes/helper_function.py

```diff
@@ -0,0 +1,17 @@
+from flask import abort, make_response, request
+
+def get_model_from_id(cls,model_id):
```

Comment:
Great helper function.  I do suggest also having one to validate the request body as well.